docs/source/user_guide/micromamba.rst

```diff
@@ -198,4 +199,77 @@ To install such a file with ``micromamba``, just pass the ``-f`` flag again:
 
 .. code::
 
-  $ micromamba create -n xtensor -c conda-forge -f explicit_env.txt
+  $ micromamba create -n xtensor -f explicit_env.txt
```

Comment:
Since each package is specified by URL in an explicit spec file, we don't need to specify a channel.